libzfs.pyx

```diff
@@ -1903,11 +1904,67 @@ cdef class ZFSPool(object):
+                                        'replicas exist for the pool to continue functioning in a degraded state.',
+                PoolStatus.RESILVERING: 'One or more devices is currently being resilvered. The pool will continue '
+                                        'to function, possibly in a degraded state.',
+                PoolStatus.CORRUPT_DATA: 'One or more devices has experienced an error resulting in data '
+                                         'corruption. Applications may be affected.',
+                PoolStatus.CORRUPT_POOL: 'The pool metadata is corrupted and the pool cannot be opened.',
+                PoolStatus.VERSION_OLDER: 'The pool is formatted using a legacy on-disk format. The pool can still '
+                                          'be used, but some features are unavailable.',
+                PoolStatus.VERSION_NEWER: 'The pool has been upgraded to a newer, incompatible on-disk version. '
+                                          'The pool cannot be accessed on this system.',
+                PoolStatus.FEAT_DISABLED: 'Some supported features are not enabled on the pool. The pool can still '
+                                          'be used, but some features are unavailable.',
+                PoolStatus.UNSUP_FEAT_READ: 'The pool cannot be accessed on this system because it uses the following '
+                                            f'feature(s) not supported on this system: {self.__unsup_features()}',
+                PoolStatus.UNSUP_FEAT_WRITE: 'The pool can only be accessed in read-only mode on this system. It '
+                                             'cannot be accessed in read-write mode because it uses the following '
+                                             f'feature(s) not supported on this system: {self.__unsup_features()}',
+                PoolStatus.FAULTED_DEV_R: 'One or more devices are faulted in response to persistent errors. '
+                                          'Sufficient replicas exist for the pool to continue functioning in a '
+                                          'degraded state.',
+                PoolStatus.FAULTED_DEV_NR: 'One or more devices are faulted in response to persistent errors. '
+                                           'There are insufficient replicas for the pool to continue functioning.',
+                PoolStatus.IO_FAILURE_CONTINUE: 'One or more devices are faulted in response to IO failures.',
+                PoolStatus.BAD_LOG: 'An intent log record could not be read. Waiting for administrator intervention '
+                                    'to fix the faulted pool.',
+                PoolStatus.NON_NATIVE_ASHIFT: 'One or more devices are configured to use a non-native block size. '
+                                              'Expect reduced performance.'
+            }
+            return f'{code.name}{f" - {status_mapping.get(code.value)}" if code.value in status_mapping else ""}'
```

Comment:
It seems returning code.name here only makes things harder for API users, it can be retrieve separately from status_code